### DISCUSSION TITLE
Oauth para autorização de acesso em conta bancária

### DISCUSSION THREAD

**Discussion by starkoithi** [frontendbr/forum#2401] [Perguntas e Respostas]
Olá, tudo bem? Me chamo Koithi, sou UX Designer e estou trabalhando em um projeto de correspondente bancário. 
O correspondente bancário é basicamente um autônomo ou empresa autorizada a comercializar alguns produtos dos bancos. 

Conseguimos junto ao banco uma forma de acesso a conta bancária **(OAUTH)** onde é possível consultar e contratar empréstimos para o usuário de uma forma muito simples e rápida. Porém, 99% dos usuários não utiliza essa forma de acesso, preferindo só enviar conta corrente e agência e ter uma simulação de empréstimo mais demorada.

Gostaria de saber de vocês se já passaram por situação parecida e se conseguiram resolver de alguma maneira. As vezes sinto que posso fazer a landing page perfeita, passando todas as informações de segurança que mesmo assim o usuário vai ficar com medo de ter a conta invadida.

**Comment by Leenaa1255** [comment]
Hi guys
O OAuth é um protocolo seguro e confiável para autorizar o acesso a contas bancárias, garantindo a proteção dos dados do usuário e evitando exposição a riscos de segurança[.](https://ffstylishname.com
)
